src/client/style/oi.css

```diff
@@ -429,3 +429,12 @@ h3 {
 .points .table td .oi-icon {
   margin-top: 9px;
 }
+
+#overlay:hover {
```

Comment:
Since it's a custom class, describe it .overlay and not #overlay (which is used for id)

Moreover, to better identify custom classes, you must write it oi-overlay (oi being the shortcut for optimistik)